apps/service_persist/test/persist/compactor/presto_int_test.exs

```diff
@@ -0,0 +1,70 @@
+defmodule Persist.Compactor.PrestoIntTest do
+  use ExUnit.Case
+  use Divo
+  require Temp.Env
+
+  @prestige [
+    url: "http://localhost:8080",
+    user: "testing",
+    catalog: "hive",
+    schema: "default"
+  ]
+
+  Temp.Env.modify([
+    %{
+      app: :service_persist,
+      key: Persist.Compactor.Presto,
+      set: [
+        prestige: @prestige
+      ]
+    }
+  ])
+
+  @moduletag integration: true, divo: true
+
+  test "should compact a table in presto" do
+    persist =
+      Load.Persist.new!(
+        id: "persist-1",
+        dataset_id: "ds1",
+        subset_id: "sb1",
+        source: "topic-a",
+        destination: "table_a"
+      )
+
+    session = Prestige.new_session(@prestige)
+    {:ok, _} = Prestige.execute(session, "CREATE TABLE table_a(name varchar, age integer)")
+
+    1..10
+    |> Enum.each(fn _ ->
+      {:ok, _} =
+        Prestige.execute(session, "INSERT INTO table_a(name, age) values(#{generate_row()})")
+    end)
+
+    Prestige.execute!(session, "select * from table_a")
+    |> Prestige.Result.as_maps()
+
+    assert 10 == number_of_s3_files()
```

Comment:
do you have some cruft from a previous test in here that's bumping up the value by one or is there some additional metadata file that's causing your test to fail?

Reply:
Not sure, still working on understanding the problem.